batavia/modules/dis.js

```diff
@@ -84,6 +84,7 @@ batavia.modules.dis = {
         batavia.modules.dis.def_op('DUP_TOP_TWO', 5);
 
         batavia.modules.dis.def_op('NOP', 9);
+        batavia.modules.dis.NOP = 9;
```

Comment:
We need the constant `9` accessible as well, so that we can check if an opcode coming in is equal to this. The `def_op` function stores it in an object, which is a little more annoying to get at.

We do a similar thing elsewhere (in particular, `EXTENDED_ARG` at the bottom).